indra/ontology/app/app.py

```diff
@@ -59,6 +59,15 @@ def get_component_label():
         **{k: v for k, v in request.json.items() if k in kwargs}))
 
 
+@app.route('/get_xrefs', methods=['GET'])
+def get_xrefs():
+    ont = request.json.get('ontology')
+    ontology = ontologies.get(ont)
+    kwargs = ('ns', 'id')
+    return jsonify(ontology.get_mappings(
+        **{k: v for k, v in request.json.items() if k in kwargs}))
```

Comment:
This function takes a single `ns` and `id` argument so the use of the dict iteration seems odd, why not just use
```
ontology.get_mappings(request.json['ns'], request.json['id'])
```

Reply:
Huh, looks like I started this pattern in prior endpoints, and I can't recall why it would have made sense to do this, other than for optional arguments...